core/src/main/scala/org/bitcoins/core/wallet/transactions/TransactionDb.scala

```diff
@@ -1,16 +1,9 @@
-package org.bitcoins.wallet.models
+package org.bitcoins.core.wallet.transactions
```

Comment:
same here, since it's part of the API we should probably have it in

`org.bitcoins.core.api.wallet` I think?